### DISCUSSION TITLE
Export ifc to json or include parameter to gltf

### DISCUSSION THREAD

**Discussion by Undianer** [IfcOpenShell/IfcOpenShell#2180] [Q&A]
I would like to know if its possible to export an ifc-file to a json-file. The reason is, that I use a viewer which is able to read its own format or a gltf format and links a json file to the mesh in order to link the metadata to the objects.
I tried to export it as ifcjson, but I don't get any options to do that. It just saves the ifc-file and overwrites the original. Speaking of it. Is it possible to save the ifc to a new file or do I have to copy the original before working with it?

Thanks in advance!

**Comment by Moult** [comment]
Have you tried Save-As in the project panel and then changing the file extension to .ifcjson?
